dist/profile/manifests/buildmaster.pp

```diff
@@ -275,6 +276,13 @@
     require => Package['httpd'],
   }
 
+  file { "${docroot}/empty.json" :
+    ensure  => file,
+    content => '{}',
+    mode    => '0755',
```

Comment:
Is there any reasons why empty.json is an executable file?
I think 0644 is more appropriated